tools/packaging/packaging.psm1

```diff
@@ -1792,185 +2183,182 @@ function New-ILNugetPackage
-                # This is for enabling applications that hosts PowerShell to ship the built-in modules.
-
-                $winBuiltInModules = @(
-                    "CimCmdlets",
-                    "Microsoft.PowerShell.Diagnostics",
-                    "Microsoft.PowerShell.Host",
-                    "Microsoft.PowerShell.Management",
-                    "Microsoft.PowerShell.Security",
-                    "Microsoft.PowerShell.Utility",
-                    "Microsoft.WSMan.Management",
-                    "PSDiagnostics"
-                )
-
-                $unixBuiltInModules = @(
-                    "Microsoft.PowerShell.Host",
-                    "Microsoft.PowerShell.Management",
-                    "Microsoft.PowerShell.Security",
-                    "Microsoft.PowerShell.Utility"
-                )
-
-                $winModuleFolder = New-Item (Join-Path $contentFolder "runtimes\win\lib\$script:netCoreRuntime\Modules") -ItemType Directory -Force
-                $unixModuleFolder = New-Item (Join-Path $contentFolder "runtimes\unix\lib\$script:netCoreRuntime\Modules") -ItemType Directory -Force
-
-                foreach ($module in $winBuiltInModules) {
-                    $source = Join-Path $WinFxdBinPath "Modules\$module"
-                    Copy-Item -Path $source -Destination $winModuleFolder -Recurse -Force
-                }
+            $winModuleFolder = New-Item (Join-Path $contentFolder "runtimes\win\lib\$script:netCoreRuntime\Modules") -ItemType Directory -Force
+            $unixModuleFolder = New-Item (Join-Path $contentFolder "runtimes\unix\lib\$script:netCoreRuntime\Modules") -ItemType Directory -Force
 
-                foreach ($module in $unixBuiltInModules) {
-                    $source = Join-Path $LinuxFxdBinPath "Modules\$module"
-                    Copy-Item -Path $source -Destination $unixModuleFolder -Recurse -Force
-                }
+            foreach ($module in $winBuiltInModules) {
+                $source = Join-Path $WinFxdBinPath "Modules\$module"
+                Copy-Item -Path $source -Destination $winModuleFolder -Recurse -Force
+            }
 
-                Write-Log "Copied the built-in modules to contentFiles for the SDK package"
+            foreach ($module in $unixBuiltInModules) {
+                $source = Join-Path $LinuxFxdBinPath "Modules\$module"
+                Copy-Item -Path $source -Destination $unixModuleFolder -Recurse -Force
             }
 
-            #region nuspec
-            # filed a tracking bug for automating generation of dependecy list: https://github.com/PowerShell/PowerShell/issues/6247
-            $deps = [System.Collections.ArrayList]::new()
+            Write-Log "Copied the built-in modules to contentFiles for the SDK package"
+        }
+
+        #region nuspec
+        # filed a tracking bug for automating generation of dependecy list: https://github.com/PowerShell/PowerShell/issues/6247
```

Comment:
Is this code same as L1738? Can we refactor to reuse common code?

Reply:
The `New-ILNugetPackage` function is no longer used, however I didn't want to remove it during testing.  But I'll go ahead and remove it now.